source/glossary/index.html.md.erb

```diff
@@ -0,0 +1,31 @@
+---
+title: Glossary
+last_reviewed_on: 2020-01-15
+review_in: 6 months
+weight: 2
+---
+
+# <%= current_page.data.title %>
+
+
+
+### Component
```

Comment:
I would expect a second level heading to follow a first level, is there a reason you didn't do this?

Reply:
yeah, felt it just didn't look right. Font too big and too much whitespace between. Would maybe prefer to use an html definition list or something like that but tried and doesn't look like this markdown processor supports the syntax.


![image](https://user-images.githubusercontent.com/960806/72599988-ac378f80-390a-11ea-9530-5c588f380c68.png)
compared to 
![image](https://user-images.githubusercontent.com/960806/72600137-f9b3fc80-390a-11ea-8900-314d9c52839a.png)